rewards/controllers.go

```diff
@@ -0,0 +1,61 @@
+package rewards
+
+import (
+	"errors"
+	"net/http"
+
+	appctx "github.com/brave-intl/bat-go/utils/context"
+	"github.com/brave-intl/bat-go/utils/handlers"
+	"github.com/brave-intl/bat-go/utils/inputs"
+	"github.com/brave-intl/bat-go/utils/logging"
+)
+
+// GetParametersHandler - handler to get reward parameters
+func GetParametersHandler(service *Service) handlers.AppHandler {
+	return handlers.AppHandler(func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		// get context from request
+		ctx := r.Context()
+
+		var (
+			currencyInput = r.URL.Query().Get("currency")
+
+			// response structure
+			parameters *Parameters
+		)
+
+		if currencyInput == "" {
```

Comment:
Does this mean that if a currency isn't provided by the service that we're going to default to USD?

Reply:
Correct.